plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/exporter/DataExporterCSV.java

```diff
@@ -101,6 +103,10 @@ public void init(IStreamDataExporterSite site) throws DBException
         if (ROW_DELIMITER_DEFAULT.equalsIgnoreCase(this.rowDelimiter.trim())) {
             this.rowDelimiter = GeneralUtils.getDefaultLineSeparator();
         }
+        this.lineFeedEscapeString = CommonUtils.toString(properties.get(PROP_LINE_FEED_ESCAPE_STRING), null)
```

Comment:
NPE?
CommonUtils.toString(properties.get(PROP_LINE_FEED_ESCAPE_STRING), null) return null if property is missing